drop make

LinusU opened the pull request:
ref #640 #642 

Ping @zbjornson

zbjornson commented:
I like! Travis doesn't know how to benchmark (https://travis-ci.org/Automattic/node-canvas/jobs/85271371#L3305); could you update https://github.com/Automattic/node-canvas/blob/master/.travis.yml#L24 to use the npm script instead?

LinusU commented:
Fixed, also changed `server` to `test-server` since `npm` gave a warning.

> npm WARN package.json canvas@1.2.10 scripts['server'] should probably be scripts['start'].

LinusU commented:
Okay, I'm merging this now
